app/assets/fonts/glyphs.txt

```diff
@@ -0,0 +1 @@
+ !#%&(),-./0123456789:;?@ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz «»¿ÀÁÈÉÊÎÓÚàáâãçèéêëíîïñóôùúû ‑—‘’“”…‹中体文简
```

Comment:
Is there a way we could do this dynamically at deploy time? This seems like an annoying thing to fail the build on

Reply:
I do think it's burdensome, yes. But realistically I also think this is something that will almost never happen.